ResidentInformationApi.Tests/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{

```

Comment:
I think this can be deleted?

Reply:
The whole file?

Reply:
I'm not sure, is it used?

Reply:
Not sure, but it looks like it's not as it's got transaction API init?

Reply:
It looks like the other APIs have it. I guess it's fine.

Reply:
I've just tested it without it and it works, will remove it as i can be confusing

Reply:
Oh okay, I'll leave it then